### ISSUE TITLE
Vagrant Up fails on mariadb config

### ISSUE THREAD

**Issue by LMDGROUP** [puphpet/puphpet#2317]
I have attempted a few different configurations however all seems to end with the following error.  

````    
==> machine20: Notice: /Stage[main]/Mysql::Server::Root_password/File[/root/.my.cnf]: Dependency Mysql_user[root@localhost] has failures: true
==> machine20: Notice: /Stage[main]/Mysql::Server::Root_password/File[/root/.my.cnf]: Dependency Apt_key[Add key: 7BD9BF62 from Apt::Source nginx] has failures: true
==> machine20: Warning: /Stage[main]/Mysql::Server::Root_password/File[/root/.my.cnf]: Skipping because of failed dependencies
==> machine20: Notice: /Stage[main]/Mysql::Server/Anchor[mysql::server::end]: Dependency Mysql_user[root@localhost] has failures: true
==> machine20: Notice: /Stage[main]/Mysql::Server/Anchor[mysql::server::end]: Dependency Apt_key[Add key: 7BD9BF62 from Apt::Source nginx] has failures: true
==> machine20: Warning: /Stage[main]/Mysql::Server/Anchor[mysql::server::end]: Skipping because of failed dependencies
==> machine20: Info: Class[Puphpet_ruby]: Unscheduling all events on Class[Puphpet_ruby]
==> machine20: Error: Could not prefetch mysql_grant provider 'mysql': Command mysql is missing
==> machine20: Notice: /Stage[main]/Puphpet_mariadb/Mysql_grant[dbuser@localhost/*.*]: Dependency Mysql_database[dbname] has failures: true
==> machine20: Notice: /Stage[main]/Puphpet_mariadb/Mysql_grant[dbuser@localhost/*.*]: Dependency Mysql_user[root@localhost] has failures: true
==> machine20: Notice: /Stage[main]/Puphpet_mariadb/Mysql_grant[dbuser@localhost/*.*]: Dependency Apt_key[Add key: 7BD9BF62 from Apt::Source nginx] has failures: true
==> machine20: Notice: /Stage[main]/Puphpet_mariadb/Mysql_grant[dbuser@localhost/*.*]: Dependency Mysql_user[dbuser@localhost] has failures: true
==> machine20: Warning: /Stage[main]/Puphpet_mariadb/Mysql_grant[dbuser@localhost/*.*]: Skipping because of failed dependencies
==> machine20: Info: Class[Puphpet_mariadb]: Unscheduling all events on Class[Puphpet_mariadb]
==> machine20: Info: Stage[main]: Unscheduling all events on Stage[main]
==> machine20: Error: Could not find a suitable provider for mysql_user
==> machine20: Error: Could not find a suitable provider for rvm_system_ruby
==> machine20: Error: Could not find a suitable provider for rvm_gem
==> machine20: Error: Could not find a suitable provider for mysql_database
==> machine20: Info: Creating state file /opt/puppetlabs/puppet/cache/state/state.yaml
==> machine20: Notice: Applied catalog in 12.80 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
````    

Here are the confg file details...

````    vagrantfile:
    target: local
    vm:
        provider:
            local:
                box: puphpet/ubuntu1404-x64
                box_url: puphpet/ubuntu1404-x64
                box_version: '0'
                chosen_virtualizer: virtualbox
                virtualizers:
                    virtualbox:
                        modifyvm:
                            natdnshostresolver1: false
                        showgui: 0
                    vmware:
                        numvcpus: 1
                    parallels:
                        use_linked_clone: 0
                        check_guest_tools: 0
                        update_guest_tools: 0
                machines:
                    vflm_op493hmcbzac:
                        id: machine20
                        hostname: machine20.puphpet
                        network:
                            private_network: 192.168.56.101
                            forwarded_port:
                                vflmnfp_lk3dyuq4i5kv:
                                    host: '6468'
                                    guest: '22'
                        memory: '2048'
                        cpus: '1'
        provision:
            puppet:
                manifests_path: puphpet/puppet/manifests
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
        synced_folder:
            vflsf_ko4akze7lfvb:
                source: /Users/josephlyons/Sites/20.dev
                target: /var/www/20.dev
                sync_type: nfs
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
        post_up_message: ''
    ssh:
        host: 'false'
        port: 'false'
        private_key_path: 'false'
        username: vagrant
        guest_port: 'false'
        keep_alive: '1'
        forward_agent: 'false'
        forward_x11: 'false'
        shell: 'bash -l'
        insert_key: 'false'
    vagrant:
        host: detect
    proxy:
        http: ''
        https: ''
        ftp: ''
        no_proxy: ''
server:
    install: '1'
    packages: {  }
users_groups:
    install: '1'
    groups: {  }
    users:
        - 'josephlyons:root'
locale:
    install: '1'
    settings:
        default_locale: en_US.UTF-8
        locales:
            - en_GB.UTF-8
            - en_US.UTF-8
        timezone: UTC
firewall:
    install: '1'
    rules:
        fr_e5hdsud5ud0r:
            port:
                - '80'
            priority: '100'
            proto: tcp
            action: accept
cron:
    install: '1'
    jobs: {  }
nginx:
    install: '1'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_p9jrbbkt52x1:
            server_name: dev20.dev
            server_aliases:
                - www.dev20.dev
            www_root: /var/www/dev20.dev
            listen_port: '80'
            index_files:
                - index.html
                - index.htm
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_120kpluxmzez:
                    location: /
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                    proxy: ''
                    proxy_redirect: ''
                nxvl_79mx5wmezi74:
                    location: '~ \.php$'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
                    set:
                        - '$path_info $fastcgi_path_info'
                    proxy: ''
                    proxy_redirect: ''
    proxies: {  }
apache:
    install: '0'
    settings:
        version: 2.4
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - proxy_fcgi
        - rewrite
    vhosts:
        av_ksuyh4eydak1:
            servername: awesome.dev
            serveraliases:
                - www.awesome.dev
            docroot: /var/www/awesome
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_aqijnju3fj3l:
                    path: /var/www/awesome
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_12c4tt6gs6gc:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
letsencrypt:
    install: '1'
    settings:
        email: ''
    domains: {  }
php:
    install: '1'
    settings:
        version: '70'
    modules:
        php:
            - cli
            - intl
            - mysql
            - fpm
        pear: {  }
        pecl: {  }
    ini:
        display_errors: 'On'
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        date.timezone: UTC
        cgi.fix_pathinfo: '0'
    fpm_ini:
        error_log: /var/log/php-fpm.log
    fpm_pools:
        phpfp_cahd4st9elo6:
            ini:
                prefix: www
                listen: '127.0.0.1:9000'
                security.limit_extensions: .php
                user: www-user
                group: www-data
    composer: '1'
    composer_home: ''
xdebug:
    install: '0'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
blackfire:
    install: '0'
    settings:
        server_id: ''
        server_token: ''
        agent:
            http_proxy: ''
            https_proxy: ''
            log_file: stderr
            log_level: '1'
        php:
            agent_timeout: '0.25'
            log_file: ''
            log_level: '1'
xhprof:
    install: '0'
wpcli:
    install: '1'
    version: v0.21.1
drush:
    install: '0'
    version: 8.0.5
ruby:
    install: '1'
    versions:
        rv_4zxlj5yn9anf:
            default: '1'
            bundler: '1'
            version: 1.9.3
            gems:
                - deep_merge
                - activesupport
                - vine
python:
    install: '1'
    packages: {  }
    versions: {  }
nodejs:
    install: '0'
    settings:
        version: '5'
    npm_packages: {  }
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings: {  }
    server_ini:
        hhvm.server.host: 127.0.0.1
        hhvm.server.port: '9000'
        hhvm.log.use_log_file: '1'
        hhvm.log.file: /var/log/hhvm/error.log
    php_ini:
        display_errors: 'On'
        error_reporting: '-1'
        date.timezone: UTC
mariadb:
    install: '1'
    settings:
        version: '10.1'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mariadbnu_y7pessbj1371:
            name: dbuser
            password: '123'
    databases:
        mariadbnd_a0w2us0cdpja:
            name: dbname
            sql: ''
    grants:
        mariadbng_g4e1aq25vxeb:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
mysql:
    install: 0
    settings:
        version: '5.6'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mysqlnu_xkanbh2cl81e:
            name: dbuser
            password: '123'
    databases:
        mysqlnd_f8smmipi9cos:
            name: dbname
            sql: ''
    grants:
        mysqlng_qcr6817epi4h:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
postgresql:
    install: '0'
    settings:
        global:
            encoding: UTF8
            version: '9.4'
        server:
            postgres_password: '123'
    databases: {  }
    users: {  }
    grants: {  }
    adminer: 0
mongodb:
    install: '0'
    settings:
        bind_ip: 127.0.0.1
        port: '27017'
    globals:
        version: 2.6.0
    databases: {  }
redis:
    install: '1'
    settings:
        port: '6379'
sqlite:
    install: '0'
    adminer: 0
    databases: {  }
mailhog:
    install: '0'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '8025'
        path: /usr/local/bin/mailhog
beanstalkd:
    install: '0'
    settings:
        listenaddress: 0.0.0.0
        listenport: '11300'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: 0
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '0'
    settings:
        version: 2.3.1
        java_install: true
    instances:
        esi_j4xh7nwpc7wd:
            name: es-01
solr:
    install: '0'
    settings:
        version: 4.10.2
        port: '8984'
````

**Comment by MUTOgen** [comment]
Have the same issue too, but with mysql.
How to solve it?

**Comment by jtreminio** [comment]
Fixed in latest release.

**Comment by martisj** [comment]
I received the same error message now. With a config I download 20 minutes ago. 3.nov @ 12:43 CET.

Config: 
```yaml
vagrantfile:
    target: local
    vm:
        provider:
            local:
                box: puphpet/ubuntu1404-x64
                box_url: puphpet/ubuntu1404-x64
                box_version: '0'
                chosen_virtualizer: virtualbox
                virtualizers:
                    virtualbox:
                        modifyvm:
                            natdnshostresolver1: false
                        showgui: 0
                    vmware:
                        numvcpus: 1
                    parallels:
                        linked_clone: 0
                        check_guest_tools: 0
                        update_guest_tools: 0
                machines:
                    vflm_0wu0ikd8fey6:
                        id: ubuntu_php_apache
                        hostname: machine1.puphpet
                        network:
                            private_network: 192.168.56.101
                            forwarded_port:
                                vflmnfp_fbhvo7ap6yjm:
                                    host: '6224'
                                    guest: '22'
                        memory: '512'
                        cpus: '1'
        provision:
            puppet:
                manifests_path: puphpet/puppet/manifests
                module_path:
                    - puphpet/puppet/modules
                    - puphpet/puppet/manifests
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
        synced_folder:
            vflsf_41l40gxoyebi:
                source: ~/Projects
                target: /var/www
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
        post_up_message: ''
    ssh:
        host: 'false'
        port: 'false'
        private_key_path: 'false'
        username: vagrant
        guest_port: 'false'
        keep_alive: '1'
        forward_agent: 'false'
        forward_x11: 'false'
        shell: 'bash -l'
        insert_key: 'false'
    vagrant:
        host: detect
    proxy:
        http: ''
        https: ''
        ftp: ''
        no_proxy: ''
server:
    install: '1'
    packages: {  }
users_groups:
    install: '1'
    groups: {  }
    users: {  }
locale:
    install: '1'
    settings:
        default_locale: en_US.UTF-8
        locales:
            - en_GB.UTF-8
            - en_US.UTF-8
        timezone: Europe/Oslo
firewall:
    install: '1'
    rules: {  }
cron:
    install: '1'
    jobs: {  }
nginx:
    install: '0'
    settings:
        version: present
        default_vhost: 1
        proxy_buffers: '4 256k'
        proxy_buffer_size: 128k
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_pkek5uvn1zj8:
            server_name: awesome.dev
            server_aliases:
                - www.awesome.dev
            www_root: /var/www/awesome
            listen_port: '80'
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_uqtvwi5qwe2y:
                    www_root: ''
                    location: /
                    index_files:
                        - index.html
                        - index.htm
                        - index.php
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                    proxy: ''
                    proxy_redirect: ''
                nxvl_e8o6s4kclg5d:
                    www_root: ''
                    location: '~ \.php$'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
                    set:
                        - '$path_info $fastcgi_path_info'
                    proxy: ''
                    proxy_redirect: ''
    proxies: {  }
apache:
    install: '1'
    settings:
        version: 2.4
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - proxy_fcgi
        - rewrite
    vhosts:
        av_nycr2y4decvd:
            servername: hjemtiljul.dev
            docroot: /var/www/sos/sos-server
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_efl5qz11391h:
                    path: /var/www/sos/sos-server
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_p2t6d3ttmt76:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
letsencrypt:
    install: '1'
    settings:
        email: msjastad@oslo.ddb.com
    domains: {  }
php:
    install: '1'
    settings:
        version: '5.6'
    modules:
        php:
            - cli
            - intl
            - xml
        pear: {  }
        pecl: {  }
    ini:
        display_errors: 'On'
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        date: '0'
        date.timezone: Europe/Oslo
    fpm_ini:
        error_log: /var/log/php-fpm.log
    fpm_pools:
        phpfp_a8yto4g672lc:
            ini:
                prefix: www
                listen: '127.0.0.1:9000'
                security.limit_extensions: .php
                user: www-user
                group: www-data
    composer: '1'
    composer_home: ''
xdebug:
    install: '1'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
blackfire:
    install: '0'
    settings:
        server_id: ''
        server_token: ''
        agent:
            http_proxy: ''
            https_proxy: ''
            log_file: stderr
            log_level: '1'
        php:
            agent_timeout: '0.25'
            log_file: ''
            log_level: '1'
xhprof:
    install: '1'
wpcli:
    install: '1'
    version: v0.24.1
drush:
    install: '0'
    version: 8.0.5
ruby:
    install: '1'
    versions:
        rv_xitzbonoo2qy:
            default: '1'
            bundler: '1'
            version: 2.3.1
            gems:
                - deep_merge@1.0.1
                - activesupport@4.2.6
                - vine@0.2
python:
    install: '1'
    packages: {  }
    versions:
        pyenv_681ki5lvps5t:
            version: 3.4.0
            virtualenv: '1'
nodejs:
    install: '1'
    settings:
        version: '6'
    npm_packages: {  }
hhvm:
    install: '0'
    composer: '1'
    composer_home: ''
    settings: {  }
    server_ini:
        hhvm.server.host: 127.0.0.1
        hhvm.server.port: '9000'
        hhvm.log.use_log_file: '1'
        hhvm.log.file: /var/log/hhvm/error.log
    php_ini:
        display_errors: 'On'
        error_reporting: '-1'
        date.timezone: UTC
mariadb:
    install: '1'
    settings:
        version: '10.1'
        root_password: '123'
        override_options: {  }
    adminer: '1'
    users:
        mariadbnu_km0i5aiswz6t:
            name: dbuser
            password: '123'
    databases:
        mariadbnd_uegr8p6x8021:
            name: dbname
            sql: ''
    grants:
        mariadbng_h2y85kjigfn6:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
mysql:
    install: '0'
    settings:
        version: '5.7'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mysqlnu_yylz1sq1xf02:
            name: dbuser
            password: '123'
    databases:
        mysqlnd_uqm1qnpxckjv:
            name: dbname
            sql: ''
    grants:
        mysqlng_nmcbgoigx8w6:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
postgresql:
    install: '1'
    settings:
        global:
            encoding: UTF8
            version: '9.6'
        server:
            postgres_password: '123'
    databases: {  }
    users: {  }
    grants: {  }
    adminer: '1'
mongodb:
    install: '0'
    settings:
        bind_ip: 127.0.0.1
        port: '27017'
    globals:
        version: 2.6.0
    databases: {  }
redis:
    install: '1'
    settings:
        port: '6379'
sqlite:
    install: '0'
    adminer: 0
    databases: {  }
mailhog:
    install: '1'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '8025'
        path: /usr/local/bin/mailhog
beanstalkd:
    install: '0'
    settings:
        listenaddress: 0.0.0.0
        listenport: '11300'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: 0
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '0'
    settings:
        version: 2.3.1
        java_install: true
    instances:
        esi_91e1y87n6u4i:
            name: es-01
solr:
    install: '0'
    settings:
        version: 5.5.2
        port: '8984'

```

**Comment by phlisg** [comment]
I'm joining you on this one @martisj ... 

TL;DR: might be network adapter problem

But I may have a thought about what's going wrong: on my Windows machine, I have a stupid Killer Ethernet adapter on my Mobo, and before I disabled the "Killer management" in the properties on all VirtualBox + real ethernet adapters, I couldn't even connect via SSH to any vagrant Puphpet machine (Homestead improved, no problems though).

After I've disabled this Killer management thingy, it went on installing, but stumbled upon installing any database, except SQLite. I have some strange errors in the very long output:

`Error: Facter: error while resolving custom fact "nodejs_latest_version": end of file reached`

That happens right after `info: loading facts`.

Follows these strange warnings:

```
Warning: The key 'shib2' is declared more than once at /tmp/vagrant-puppet/modules-ac6b1eda86798512cb3d4e54a375ea88/apache/manifests/params.pp:251:21

Warning: Unknown variable: 'ourpackageversion'. at /tmp/vagrant-puppet/modules-ac6b1eda86798512cb3d4e54a375ea88/beanstalkd/manifests/init.pp:85:8

Warning: Scope(Apt::Source[apt.postgresql.org]): $include_src is deprecated and will be removed in the next major release, please use $include => { 'src' => false } instead

Warning: Scope(Apt::Source[apt.postgresql.org]): $key_source is deprecated and will be removed in the next major release, please use $key => { 'source' => https://www.postgresql.org/media/keys/ACCC4CF8.asc } instead.

Warning: Scope(Apt::Key[Add key: B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8 from Apt::Source apt.postgresql.org]): $key_source is deprecated and will be removed in the next major release. Please use $source instead.
```

At some point it reaches the Databases section and it tries to install MariaDB, I get this:

```
Error: Execution of '/usr/bin/apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 199369E5404BD5FC7D2FE43BCBCB082A1BB943DB' returned 2: Executing: /tmp/tmp.xyM1uwwKbt/gpg.1.sh --keyserver
hkp://keyserver.ubuntu.com:80
--recv-keys
199369E5404BD5FC7D2FE43BCBCB082A1BB943DB
gpg: requesting key 1BB943DB from hkp server keyserver.ubuntu.com
?: [fd 4]: read error: Connection reset by peer
gpgkeys: key 199369E5404BD5FC7D2FE43BCBCB082A1BB943DB partially retrieved (probably corrupt)
gpg: no valid OpenPGP data found.
gpg: key 1BB943DB: public key "MariaDB Package Signing Key <package-signing-key@mariadb.org>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

Follows a little "hanging" moment, after which a huge list of `Dependency xxx has failures: true` show up, to finally end up like you, @martisj 

Editing the YAML and disabling all the databases except SQLite works.

I just bought a new Ethernet card from Intel (which should be incredibly compatible with everything...), and I'm stuck at the same place.

I tried on my computer at work (a Windows machine as well), **I don't have any of the warning/notices mentionned above,** it installs itself flawlessly... The problem is, Homestead works. Why not PuPHPet?

**Comment by maxttor** [comment]
I have the same issue with mariadb. How to fix it?

**Comment by maxttor** [comment]
@jtreminio the issue is relevant now
